rsconnect/api.py

```diff
@@ -667,19 +727,14 @@ def setup_client(self, cookies: Optional[CookieJar] = None, **kwargs: object):
         else:
             raise RSConnectException("Unable to infer Connect client.")
 
-    @property
-    def state(self):
-        return self._d
-
-    def get(self, key: str, *args: Any, **kwargs: Any):
-        return kwargs.get(key) or self.state.get(key)
-
     def pipe(self, func: Callable[P, T], *args: P.args, **kwargs: P.kwargs):
         return func(*args, **kwargs)
 
     @cls_logged("Validating server...")
     def validate_server(
         self,
+        # TODO: In practice, these arguments are not used. Should they be removed?
+        # And same for the downstream functions that are called from this function.
```

Comment:
Yes. We can instantiate this object with everything it needs. If anyone wants to validate a different server, they can create a different object and call its `validate_server` method.

Reply:
My comment was incorrect -- these arguments are used in the downstream functions. The docstring was in the wrong function, though: it was in `validate_connect_server()` below but it documents the params for this function, so I've moved it back to this function.